### PR TITLE
add miso-soup.md

### PR DESCRIPTION
**Pull request by yakataN**
Added Miso-soup recipe.

### CODE REVIEW

File: src/miso-soup.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Miso-soup is Japanese traditional home cooking.
+
+## Ingredients
+
+- dashi stock
+- miso paste
+- tofu(optional)
+- seaweed(optional)
+- green onion(optional)
```
Comment (hallzy) [inline src/miso-soup.md:11]:
```suggestion
- dashi stock
- miso paste
- tofu (optional)
- seaweed (optional)
- green onion (optional)
```

Are there amounts that we would need for this?

File: src/miso-soup.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Directions
+
+1. put about half of the water in the pot and bring the water to a boil.
```
Comment (hallzy) [inline src/miso-soup.md:15]: Half of what amount? Is there a specific amount of water needed for this recipe?

**Review by hallzy — CHANGES_REQUESTED** [review]

**Review by hallzy — CHANGES_REQUESTED** [review]
Do you have a picture of this dish that we can use?

File: src/miso-soup.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Miso-soup
+
+Miso-soup is Japanese traditional home cooking.
+
```
Comment (Knarkzel) [inline src/miso-soup.md:4]: Missing following:

```
- ⏲️ Prep time: 10 min
- 🍳Cook time: 30 min
- 🍽️ Servings: 4
```

**Pull request merged**